dbt/adapters/athena/connections.py

```diff
@@ -163,6 +163,9 @@ def inner() -> AthenaCursor:
                 cache_size=cache_size,
                 cache_expiration_time=cache_expiration_time,
             )
+
+            log.info(f"Athena query ID {query_id}")
```

Comment:
```suggestion
            logger.debug(f"Athena query ID {query_id}")
```

Reply:
@WarFox could you please use `logger` and not log, and the use the `debug` mode.
Using info severity will lead to having the queries ids printed in the stout in `dbt run`...

Reply:
Thank you, it is done now